From cbb75bbb49590ab60f5987ab0c9d49ca4ba2ed6b Mon Sep 17 00:00:00 2001
From: Alexander Grund <Alexander.Grund@tu-dresden.de>
Date: Fri, 2 Oct 2020 11:00:39 +0200
Subject: [PATCH] Define the FP register size on PPC to be double

According to GLIBC headers those are in fact double (8 Byte) not
long double (16 Byte)
---
 include/libunwind-ppc64.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/libunwind-ppc64.h b/include/libunwind-ppc64.h
index 9944628da..fed478bed 100644
--- a/include/libunwind-ppc64.h
+++ b/include/libunwind-ppc64.h
@@ -72,7 +72,7 @@ typedef uint64_t unw_word_t;
 typedef int64_t unw_sword_t;
 #endif
 
-typedef long double unw_tdep_fpreg_t;
+typedef double unw_tdep_fpreg_t;
 
 /*
  * Vector register (in PowerPC64 used for AltiVec registers)
